### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bitfort**
Add a basic contributors document.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributing to MLBox
+
+Thank you for your interest in contributing to MLBox! The best way to get started is to make sure you signed the 
+Tensorflow CLA (https://cla.developers.google.com/clas), soon to be replaced by a mlperf/mlcommons CLA.
```
Comment (sub-mod) [inline CONTRIBUTING.md:4]: Isn't this called Google CLA ?

Comment (bitfort) [reply CONTRIBUTING.md]: Yes -- Fixed.

**Comment by sub-mod** [comment]
Can you rebase the PR

**Comment by bitfort** [comment]
Rebasing as a separate PR.

**Pull request closed without merging**